iis-schemas/src/main/avro/eu/dnetlib/iis/referenceextraction/community/DocumentToConceptId.avdl

```diff
@@ -0,0 +1,13 @@
+@namespace("eu.dnetlib.iis.referenceextraction.community.schemas")
```

Comment:
General comment: comments in `*.avdl` files seem to be not applying any convention i.e. some fields are documented, others are not. We could agree on a convention for avdl files e.g. documenting all fields together with reference to schema for foreign keys.

Reply:
I am going to add the following comment in the upcoming commit:
```suggestion
//      concept ID, foreign key: Concept.id
        string conceptId;
```